backend/app/interior/schemas/mappers.py

```
@@ -165,20 +165,46 @@ def domain_to_user_library_interior(
+        else:
+            full_image_url = style.image_url
 
     return StyleInfo(
         style_id=style.id,
         name=style.name,
         description=style.description,
+        image_url=full_image_url,
     )
 
 
 def interior_type_to_style_info_response(style: "InteriorType") -> dict:
```

Ruff (0.12.2)

Undefined name `InteriorType`

(F821)